ansible/roles/kong-api/defaults/main.yml

```diff
@@ -1037,6 +1037,24 @@ kong_apis:
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
 
+  - name: userExistenceApiV2
```

Comment:
Please add it as per alphabet order and not random

Reply:
@keshavprasadms : added according to alphabet-order